src/components/message/message.spec.js

```diff
@@ -77,7 +77,18 @@ describe('Message', () => {
   describe('when in classic mode', () => {
     describe('when rendered', () => {
       it('should match the snapshot', () => {
-        OptionsHelper.colors.forEach((variant) => {
+        const colors = [
+          'default',
+          'error',
+          'help',
+          'info',
+          'maintenance',
+          'new',
+          'success',
+          'warning'
+        ];
```

Comment:
I think that `disabled` shouldn't be added to the colors helper, as it's used by multiple components through the Application. Adding it would allow to choose that color in stories in which we are using this helper.